mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -20,6 +20,7 @@ CLASS sg net/minecraft/entity/Entity
 	FIELD N prevRenderY D
 	FIELD O prevRenderZ D
 	FIELD P stepHeight F
+	FIELD Q noClip Z
```

Comment:
Possibly `ignoresCollision` to avoid "no" in a boolean field name?

I'm aware noClip is a common term but I've gotten the impression negatives like "no" should be avoided in boolean field names, and in the event someone isn't familiar with the term "no clip" it would be more descriptive.